### PR TITLE
fix / Enable all trading pairs for all markets

### PR DESCRIPTION
**Pull request by michaelc9**
**Before submitting this PR, please make sure**:
- [x ] Your code builds clean without any errors or warnings
- [x ] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story:

**A description of the changes proposed in the pull request**:
1. Enable all trading pairs for all markets

2. Remove trading related requirements (token approval, user streams, account balance, etc) when required_trading flag is False

3. Add more detailed market status in app status
4. Fix USD volume for coinbasePro get active markets
5. More safe checking in cli start up code

### CODE REVIEW

File: wings/market/ddex_market.pyx [hunk 1]
```diff
@@ -45,7 +45,7 @@ from wings.order_book cimport OrderBook
 from wings.order_book_tracker import OrderBookTrackerDataSourceType
 from wings.tracker.ddex_order_book_tracker import DDEXOrderBookTracker
 from wings.wallet.web3_wallet import Web3Wallet
-
+import pandas as pd
```
Comment (chowryan) [inline wings/market/ddex_market.pyx]: extra?

File: wings/market/market_base.pyx [hunk 2]
```diff
@@ -42,6 +42,10 @@ cdef class MarketBase(NetworkIterator):
             self.c_add_listener(event_tag.value, self.event_reporter)
             self.c_add_listener(event_tag.value, self.event_logger)
 
+    @property
+    def status_dict(self) -> Dict[str, bool]:
+        return {}
```
Comment (chowryan) [inline wings/market/market_base.pyx]: should this throw error? if you do {}.values it raises an error "TypeError: 'builtin_function_or_method' object is not iterable"

File: hummingbot/strategy/discovery/discovery.pyx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # distutils: language=c++
 import asyncio
 import logging
+import math
```
Comment (chowryan) [inline hummingbot/strategy/discovery/discovery.pyx]: not used?

File: wings/market/market_base.pyx (hunk 2 above)
Comment (michaelc9) [reply wings/market/market_base.pyx]: Calling {}.values() should return dict_values([])

**Pull request merged**